tests/testthat/_snaps/backend-postgres.md

```diff
@@ -102,7 +102,9 @@
       rows_insert(x, y, by = c("a", "b"), in_place = TRUE, conflict = "ignore",
       returning = everything(), method = "on_conflict")
     Condition
-      Error:
+      Error in `rows_insert()`:
+      ! Can't modify database table "df_x".
+      Caused by error:
```

Comment:
Should this mention that the error is (probably) produced by the database?

Reply:
I think that would be useful, but I can't come up with any good phrasing for it.

I think the ideal would be something like this, but we can't customise like that currently:

```
Error in `rows_insert()`:
 ! Can't modify table "df_x".
Caused by database error:
! Failed to fetch row: ERROR:  there is no unique or exclusion constraint matching the ON CONFLICT specification
```

Reply:
Maybe just prepend the `cnd$message` by something like `[Database error]` or `in database:`

```
Error in `rows_insert()`:
      ! Can't modify database table "df_x".
      Caused by error:
      ! [Database error] Failed to fetch row: ERROR:  there is no unique or exclusion constraint matching the ON CONFLICT specification
```

```
Error in `rows_insert()`:
      ! Can't modify database table "df_x".
      Caused by error:
      ! in database: Failed to fetch row: ERROR:  there is no unique or exclusion constraint matching the ON CONFLICT specification
```

It's not quite a nice as `Caused by database error` but should still help to understand the issue better.